ufobalapp/static/ufobalapp/js/userService.js

```diff
@@ -174,6 +174,10 @@ app.controller("auth", ["$scope", "userService", "$location", "$routeParams", "$
         });
     };
 
+    if ($scope.user.is_authorized) {
+        $scope.openProfile();
+    }
+
```

Comment:
tohle je hodně pochybný, redirektne mě to když se reloaduji skoro kdekoliv

Reply:
chci aby se user presmeroval do jeho profilu pokud si otevre parovaci stranku, ale uz je prihlasenej a sparovanej..nicmene asi se ten controller pouziva i jinde, takze je to takhle teda spatne